services/route/route.proto

```diff
@@ -17,7 +17,7 @@ message FindRouteRequest {
 }
 
 message FindRouteResponse {
-  uint32 eta_seconds = 1;
+  int32 eta_seconds = 1;
```

Comment:
@jmsktm why do we need to support negative ETA for quickstart?

Reply:
As discussed, this is to initiate the user with the scenario in which the ETA was negative, and to fix that issue by using sandboxes.